gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/apim/core/documentation/use_case/ApiCreateDocumentationPageUseCaseTest.java

```diff
@@ -64,7 +65,8 @@ class ApiCreateDocumentationPageUseCaseTest {
     private final DocumentationValidationDomainService documentationValidationDomainService = new DocumentationValidationDomainService(
         new HtmlSanitizerImpl(),
         new NoopTemplateResolverDomainService(),
-        apiCrudService
+        apiCrudService,
+        pageCrudService
     );
     AuditCrudServiceInMemory auditCrudService = new AuditCrudServiceInMemory();
     UserCrudServiceInMemory userCrudService = new UserCrudServiceInMemory();
```

Comment:
It would be nice to have some tests for the business logic of creating a swagger page + asyncapi page